packages/cli-e2e/__tests__/atomic.specs.ts

```diff
@@ -71,12 +109,10 @@ describe('ui:create:atomic', () => {
       output += chunk;
     });
 
-    if (!options.pageId) {
-      await buildTerminal
-        .when(/Use an existing hosted search page/)
-        .on('stdout')
-        .do(answerPrompt(options.promptAnswer ? options.promptAnswer : EOL))
-        .once();
+    if (options.pageName) {
+      selectPageWithCli(options, buildTerminal);
+    } else if (!options.pageId) {
+      selectDefaultWithCli(buildTerminal);
```

Comment:
Refactored for clarity:

Sure we can have 'one-stop-shop' function for setting up, but let's split the bits so we don't have to read heaps of code useless for XYZ test cases